packages/backend-function/API.md

```diff
@@ -21,11 +24,18 @@ export type FunctionProps = {
     memoryMB?: number;
     environment?: Record<string, string | BackendSecret>;
     runtime?: NodeVersion;
+    schedule?: TimeInterval | TimeInterval[];
 };
 
 // @public (undocumented)
 export type NodeVersion = 16 | 18 | 20;
 
+// @public (undocumented)
+export type Rate = `every ${number}m` | `every ${number}h` | `every day` | `every week` | `every month` | `every year`;

```

Comment:
does `${number}` allow multi-digit numbers?

Reply:
Yes it does